bots/pr/src/main/java/org/openjdk/skara/bots/pr/BackportCommand.java

```diff
@@ -172,8 +182,7 @@ private HostedRepository getTargetRepo(PullRequestBot bot, String[] parts, Print
         return potentialTargetRepo.get();
     }
 
-    private Branch getTargetBranch(String[] parts, HostedRepository targetRepo, PrintWriter reply) {
-        var targetBranchName = parts.length == 2 ? parts[1] : "master";
+    private Branch getTargetBranch(String targetBranchName, HostedRepository targetRepo, PrintWriter reply) {
```

Comment:
I don't like how the string `"master"` is now repeated in three places. Could we move that check back here? We could add the parts index as a parameter to this method and supply the full array.

Reply:
Sure. It's a good idea.